src/coffee/run.coffee

```diff
@@ -62,6 +64,12 @@ if argv.logSilent
 process.on 'SIGUSR2', -> logger.reopenFileStreams()
 process.on 'exit', => process.exit(@exitCode)
 
+removeZeroInventories = (importer, logger) ->
+  logger.info("Deleting zero inventories")
```

Comment:
This one sounds a bit quirky (deleting nothing?), how about something like 
`Deleting inventories with zero quantity on stock`
or
`Deleting empty inventories`

?

Reply:
updated here 3db330b